nengo_gui/components/ace_editor.py

```diff
@@ -6,18 +6,18 @@
 import nengo_gui.monkey
 
 class AceEditor(Component):
-    def __init__(self, viz, config, uid):
-        super(AceEditor, self).__init__(viz, config, uid)
-        self.viz = viz
+    def __init__(self, sim, config, uid):
+        super(AceEditor, self).__init__(sim, config, uid, z_order=-8)
```

Comment:
I've cleaned up the code and documented this now...  and discovered that you're completely right that AceEditor doesn't need to be in a particular sort order.  So I got rid of that setting.  :)